src/io/vtk/vtk-custom-decimal.cpp

```diff
@@ -0,0 +1,144 @@
+#include "xml-schema.h"
+
+#include <limits>
+#include <locale>
+#include <sstream>
+
+#include <xsd/cxx/ro-string.hxx>
+#include <xsd/cxx/zc-istream.hxx>
+
+using namespace std;
+
+// Parsing
```

Comment:
I think we will need doxygen style documentation for all the methods, in all the files

Reply:
Included doxygen style documentation in all files, also changed the VTKWriter.dox to be more up-to-date